backend/src/test/java/com/bakdata/conquery/integration/IntegrationTests.java

```diff
@@ -64,7 +64,7 @@
 	public Stream<DynamicNode> programmaticTests() {
 		List<Class<?>> programmatic = CPSTypeIdResolver
 			.SCAN_RESULT
-			.getClassesImplementing(ProgrammaticIntegrationTest.class.getName())
+			.getClassesImplementing(ProgrammaticIntegrationTest.This.class.getName())
```

Comment:
was ist  `This`?

Reply:
Oh, das ist noch eine abkürzung aus dem debugging. Das muss raus